src/main/java/thestonedturtle/partypanel/ui/skills/SkillPanelSlot.java

```diff
@@ -146,4 +149,14 @@ public void updateBoostedLevel(final int boostedLevel)
 		boostedLabel.setText(String.valueOf(boostedLevel));
 		boostedLabel.repaint();
 	}
+
+	public void updateSkillEXP(final int exp)
```

Comment:
Just use  `@Getter` and `@Setter` annotations on the `skillEXP` since these are not doing anything extra